msbuild/tests/Xamarin.iOS.Tasks.Tests/Xamarin.iOS.Tasks.Tests.csproj

```diff
@@ -26,6 +26,7 @@
     <Reference Include="Xamarin.iOS.Tasks" HintPath="..\..\Xamarin.iOS.Tasks\bin\$(Configuration)\netstandard2.0\Xamarin.iOS.Tasks.dll" />
     <PackageReference Include="NUnit" Version="3.12.0" />
     <PackageReference Include="NUnit.ConsoleRunner" Version="3.11.1" />
+    <PackageReference Include="NUnit.Extension.NUnitV2ResultWriter" Version="3.6.0" />
```

Comment:
is this not part of your other PR ?
if so it should not be here

Reply:
It is, and I think is better to land https://github.com/xamarin/xamarin-macios/pull/9545 which is a selfcontained fix, then land this +1 on @spouliot

Reply:
Good Catch! Thanks

Reply:
@mandel-macaque #9545 should be landed now!